Cargo.toml

```diff
@@ -2,5 +2,8 @@
 members = [
     "ironrdp",
     "ironrdp-session",
-    "ironrdp-cli"
+    "ironrdp-ui-client",
```

Comment:
What about naming it `ironrdp-client`? I feel that `-ui` is a bit too much (I would expect a GUI by default from an RDP client)

Reply:
I think since there is a cli only client it is important to differentiate a little. No major concerns. Maybe this can be done in future if we find it necessary

Reply:
Okay! In that case, could you add a `g` (ironrdp-gui-client) so that it’s clear we are talking about a graphical interface?

Reply:
fixed.